doc/guide.rst

```diff
@@ -0,0 +1,179 @@
+
+    class DividerScreen(UIScreen):
+
+        def __init__(self):
+            # Set title of the screen.
+            super().__init__(title=u"Divider")
+            self._message = 0
+
+The ``self._message`` variable will be used later to show result to a user.
+
+The screens main purpose is to present content to a user. For this we need widgets and containers.
+
+The :attr:`UIScreen.window` attribute is the most important part of the screen for the rendering,
+it contains the :class:`WindowContainer <simpleline.render.containers.WindowContainer>`
+container which is created and filed up by the :meth:`UIScreen.refresh` method.
+Everything added to this container is printed to the monitor.
+So we should override the :meth:`UIScreen.refresh` method and call the parent's version to
+prepare the container. Then we can add :ref:`widgets <widgets_label>` to the container.
+So the screen will continue like this:
+
+.. code-block:: python
+    :dedent: 0
+
+        def refresh(self, args=None):
+            # Fill the self.window attribute by the WindowContainer and set screen title as header.
+            super().refresh()
+
+            widget = TextWidget("Result: " + str(self._message))
+            self.window.add_with_separator(widget)
+
+The :meth:`WindowContainer.add_with_separator() <simpleline.render.containers.WindowContainer.add_with_separator>`
+method will print blank line after the
+:class:`TextWidget's <simpleline.render.widgets.TextWidget>` text.
+
+Now a user have the header and result printed on the screen but it would be nice to give him/her a
+hint how to use the Divider screen. The best suited component of the screen for this is the
+:class:`Prompt <simpleline.render.prompt.Prompt>`.
+The :class:`Prompt <simpleline.render.prompt.Prompt>` class is responsible to guide user by giving
+him/her the possible options he/she can write.
+
+We will set the message of the prompt inside of the :meth:`UIScreen.prompt` method. We also
+remove the default option for continue because it is doing the same as quit in case when we have
+only one screen in the application.
+
+.. code-block:: python
+    :dedent: 0
+
+        def prompt(self, args=None):
+            # Change user prompt
+            prompt = super().prompt()
+
+            # Set message to the user prompt. Give a user hint how he/she may control our application.
+            prompt.set_message("Pass numbers to divider in a format: 'num / num'")
+
+            # Remove continue option from the control. There is no need for that
+            # when we have only one screen.
+            prompt.remove_option('c')
+
+            return prompt
+
+When we are able to present our content to a user we want to have the possibility to process
+user input. For this purpose there is the :meth:`UIScreen.input` method. Input from a user is
+passed to this method and the screen may process it, discard it or return it for further
+option processing.
```

Comment:
`option processing` -> `processing` (seems like it could be fine just like that)